### DISCUSSION TITLE
Projects Beta - Ability to perform operations on table column

### DISCUSSION THREAD

**Discussion by adamcole123** [community/community#16985] [Projects and Issues]
I would either like the ability to download a project table as a CSV file, or to have an option at the bottom of a column on the table to perform mathematical operations on the data. For example. I use projects to give issues sizings to predict how long they would take. 

In order to see a sum of these sizings (and therefore predict how long the project will take), I need to first copy the table (which can be annoying if there are alot of issues in the project, since you cant just click and drag), and then paste the data into an excel file. The ability to do this from inside github would be excellent.

**Comment by Martin-Fiser** [comment]
I have the same issue. But I would like to use GraphQL, unfortunately cannot get the correct query.
